crates/macros/src/lib.rs

```diff
@@ -42,10 +42,15 @@ pub fn build(stream: TokenStream) -> TokenStream {
         Err(t) => return t.into(),
     };
 
-    let mut source = ::winrt_gen::build_windows_dir();
+    let workspace_dir = ::winrt_gen::workspace_dir();
+
+    let mut source = ::winrt_gen::build_windows_dir_at_workspace(&workspace_dir);
```

Comment:
These uses of `winrt_gen` don't need to have the leading `::`.

Reply:
done - removed ::